src/mesh/meshblock.hpp

```diff
@@ -161,6 +161,13 @@ class MeshBlock : public std::enable_shared_from_this<MeshBlock> {
   std::unique_ptr<BoundarySwarms> pbswarm;
   std::unique_ptr<MeshRefinement> pmr;
 
+  // Block connectivity information
+  std::vector<NeighborBlock> neighbors;
```

Comment:
yeah, `pbvals` duplicates this information. I think with this PR there is very little that `pbvals` still does since the `ForEachBoundary` loops depend on the new vectors of `NeighborBlocks`.

Reply:
:+1: It would be good to remove `pbvals` at some point if we can then. Maybe should coordinate with @brryan about particle comms.